src/eradiate/test_tools/regression.py

```diff
@@ -22,33 +24,44 @@ def regression_test_plots(
     ref: np.typing.ArrayLike,
     result: np.typing.ArrayLike,
     vza: np.typing.ArrayLike,
-    filename: PathLike,
     metric: tuple[str, float],
+    ref_var: t.Optional[np.typing.ArrayLike] = None,
+    result_var: t.Optional[np.typing.ArrayLike] = None,
 ) -> None:
     """
-    Create regression test report plots.
+    Create regression test report plots. Plot errorbars if both ref_var and
+    result_var are set.
 
     Parameters
     ----------
     ref : array-like
-        BRF values for the reference data
+        Variable values for the reference data
 
     result : array-like
-        BRF values for the simulation result
+        Variable values for the simulation result
 
     vza : array-like
         VZA values for plotting
 
-    filename : path-like
-        Path to the output file for the plot
-
     metric : tuple
         A tuple of the form (metric name, value) to be added to the plots.
+
+    ref_var: array-like (optional)
```

Comment:
As per the [Numpydoc spec](https://numpydoc.readthedocs.io/en/latest/format.html#parameters):
```suggestion
    ref_var: array-like, optional
```